pkgs/development/libraries/grib-api/default.nix

```diff
@@ -11,6 +11,13 @@ stdenv.mkDerivation rec{
     sha256 = "0qbj12ap7yy2rl1pq629chnss2jl73wxdj1lwzv0xp87r6z5qdfl";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://sources.debian.org/data/main/g/grib-api/1.28.0-2/debian/patches/openjpeg2.patch";
```

Comment:
This is probably transient URL and may go away with some future Debian release. File in the git repo will hopefully stay longer: https://salsa.debian.org/science-team/grib-api/raw/36e3bfa4d7e0d43b217241a73ca2cb69925e14e0/debian/patches/openjpeg2.patch

Reply:
Done